### PR TITLE
chore: clean up baseUrl

### PR DESCRIPTION
**Pull request by MarcelKnobelsdorf**
Closes #

## ✅ Checks
- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build && next-image-export-optimizer --nextConfigPath ./next.config.js",
-    "start": "http-server out -p 3000",
+    "start": "http-server out -p 3001",
```
Comment (pboeder) [inline package.json:12]: Did you commit that line intentionally?

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "@primer/octicons": "19.6.0",
     "contentlayer": "0.3.4",
     "date-fns": "2.30.0",
+    "dotenv": "16.3.1",
```
Comment (pboeder) [inline package.json:27]: peerDependency

**Review by pboeder — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (MarcelKnobelsdorf) [reply package.json:12]: Thanks for the hint. I will undo it

**Review by itrich — APPROVED** [review]
LGTM! :shipit:

**Review by pboeder — APPROVED** [review]

**Pull request merged**